docs/rules/device-details/cap_var/index.rst

```diff
@@ -1,4 +1,4 @@
-Varactors
+1.8V Accumulation-Mode MOS Varactors
 ---------
```

Comment:
The dashes here need to match the length of the title.

Reply:
fixed